### PR TITLE
Aud demisto/auto update docker staging branch 14

### PR DESCRIPTION
**Pull request by mayyagoldman**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/GSuiteAdmin/Integrations/GSuiteAdmin/GSuiteAdmin.yml [hunk 1]
```diff
@@ -250,8 +250,8 @@ script:
       name: include_child_org_units
       description: Whether to return devices from all child organizational units. If this is set to true, 'org_unit_path' must be provided.
       predefined:
-      - yes
-      - no
+      - true
```
Comment (JudahSchwartz) [inline Packs/GSuiteAdmin/Integrations/GSuiteAdmin/GSuiteAdmin.yml:253]: double check this is switched in the code. check what changed this, seems like bug-prone behavior

File: Packs/GmailSingleUser/Integrations/GmailSingleUser/GmailSingleUser.yml [hunk 2]
```diff
@@ -11,61 +11,60 @@ configuration:
   name: email
   required: true
   type: 0
-  defaultvalue:
-  additionalinfo:
+  defaultvalue: null
```
Comment (JudahSchwartz) [inline Packs/GmailSingleUser/Integrations/GmailSingleUser/GmailSingleUser.yml:14]: nulls should probably be reverted. @anas-yousef is your script making this change?

File: Packs/GSuiteAdmin/Integrations/GSuiteAdmin/GSuiteAdmin.yml (hunk 1 above)
Comment (mayyagoldman) [reply Packs/GSuiteAdmin/Integrations/GSuiteAdmin/GSuiteAdmin.yml:253]: this argument is passed to argToBoolean which excepts both true false or yes no so should be fine

File: Packs/GmailSingleUser/Integrations/GmailSingleUser/GmailSingleUser.yml (hunk 2 above)
Comment (anas-yousef) [reply Packs/GmailSingleUser/Integrations/GmailSingleUser/GmailSingleUser.yml:14]: It is done from the script

**Pull request closed without merging**